packages/opentelemetry-core/src/common/logging-error-handler.ts

```diff
@@ -15,7 +15,7 @@
  */
 
 import {
-  Logger,
+  OptionalDiagLogger,
```

Comment:
Why is Optional in the name? Isn't it obvious from the `?` in the type that it is optional? It's confusing to me to have `OptionalDiagLogger` and `Diaglogger`

Reply:
In this case yes, was in "get it done mode" yesterday when converting all function arguments -- will change to DiagLogger

Reply:
Fixed